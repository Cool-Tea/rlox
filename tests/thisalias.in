class Point{
    init(y){
        this.y = y;
    }
}
var p = Point(3);
p.x=5;
print p.x*p.y;
var q = p.init(7);
print p.x*p.y;
print q.x*q.y;