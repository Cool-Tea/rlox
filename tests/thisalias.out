15
35
35